README.md

```diff
@@ -216,7 +217,7 @@ Here’s a few common cases where you might want to try something else:
 
 * If your website is **mostly static** (for example, a portfolio or a blog), consider using [Gatsby](https://www.gatsbyjs.org/) instead. Unlike Create React App, it pre-renders the website into HTML at the build time.
 
-* If you want to use **TypeScript**, consider using [create-react-app-typescript](https://github.com/wmonk/create-react-app-typescript).
+* If you want to use **TypeScript** with `tsc` instead of `babel`, consider using [create-react-app-typescript](https://github.com/wmonk/create-react-app-typescript).
```

Comment:
I'm fine just removing this all together. The differences between Babel's TypeScript syntax and actual TypeScript syntax is dwindling, so meh. Microsoft has also shown commitment to keeping Babel up to date.

Reply:
IMO the difference between what's in this PR and the `create-react-app-typescript` library should be clarified somewhere. I realize that library is an unofficial solution but it is relatively popular.

Reply:
I'd rather see this pull request go in the direction of using `tsc` instead of Babel.